StitchCore/Sources/StitchCore/Auth/Providers/Anonymous/AnonymousCredential.swift

```diff
@@ -1,20 +1,34 @@
 import ExtendedJSON
 
 /**
- * :nodoc:
  * A credential which can be used to log in as a Stitch user
  * using the anonymous authentication provider.
  */
 public struct AnonymousCredential: StitchCredential {
+    /**
+     * The name of the provider for this credential.
+     */
     public var providerName: String
 
+    /**
+     * The type of the provider for this credential.
+     */
     public var providerType: String = "anon-user"
```

Comment:
[nit] All of these `providerType` should be constants.

Reply:
Agreed. It would be nice if these were part of an enum.